build/templates/errors.py.mako

```diff
@@ -73,6 +73,15 @@ class InvalidRepeatedCapabilityError(Exception):
         super(InvalidRepeatedCapabilityError, self).__init__('An invalid character ({0}) was found in repeated capability string ({1})'.format(invalid_character, invalid_string))
 
 
+class SelfTestFailureError(Exception):
```

Comment:
I think having both "failure" and "error" (double negative) in the name of the class is redundant. One should be enough. The irony that the second sentence I typed is redundant is not lost on me.